core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/log.go

```diff
@@ -58,16 +58,23 @@ func logID(l logpoller.Log) string {
 }
 
 // blockStatistics returns the latest block number from the given logs, and a map of unique block numbers
-func blockStatistics(logs ...logpoller.Log) (int64, map[int64]bool) {
+func (b *logBuffer) blockStatistics(logs ...logpoller.Log) (int64, map[int64]bool, map[int64]bool) {
```

Comment:
nit. can we name this function better and move to buffer.go file

Reply:
Moved this into buffer.go, but not sure what a better name would be - any suggestions? 😄